config/commandFormat.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const Joi = require('joi');
+
+const HABITAT_MODE = Joi
+    .string().valid([
+        'remote',
+        'local'
+    ])
+    .description('Mode of the Habitat command')
+    .example('remote');
+
+const HABITAT_PACKAGE = Joi
+    .string()
+    .description('Package of the Habitat command')
+    .example('core/git/2.14.1');
+
+const HABITAT_BINARY = Joi
+    .string()
+    .description('Binary of the Habitat command')
+    .example('git');
+
+const SCHEMA_HABITAT = Joi.object()
+    .keys({
+        mode: HABITAT_MODE,
+        package: HABITAT_PACKAGE,
+        binary: HABITAT_BINARY
+    })
+    .requiredKeys('mode', 'package', 'binary');
+
+const DOCKER_IMAGE = Joi
+    .string()
+    .description('Image of the Docker command')
+    .example('node:1.2.3');
+
+const SCHEMA_DOCKER = Joi.object()
+    .keys({
+        image: DOCKER_IMAGE
```

Comment:
I think we should add `binary` as same as the schema of habitat, as an optional field and default "".
Some docker image have `entrypoint` or `command` set and others not.
When users want to run a SD command that uses latter one, they need to specify a command name explicitly.
> $ sd-cmd namespace/my-node node -v

If commands have binary field and a value has set, "node" in this case, users can use this command with the same parameter than other commands.
> $ sd-cmd namespace/my-node -v

Reply:
Spec. proposal:
https://github.com/screwdriver-cd/screwdriver/pull/826

Reply:
This proposal has been merged.
I'll rename and add these properties (habitat.command, docker.command).

Reply:
done.